### PR TITLE
feat: add about page

### LINKED ISSUE #57: feat: about page

### PR DESCRIPTION
**Pull request by ssarkar551**

### CODE REVIEW

**Comment by ssarkar551** [comment]
Added About page
image

**Review by AyanavaKarmakar — CHANGES_REQUESTED** [review]
Add stuff from https://github.com/TypeSafe-Travellers/App/blob/dev/README.md

Skip the "About" & "Get Started" sections tho

File: src/pages/about.tsx [hunk 1]
```diff
@@ -1,5 +1,64 @@
...
+          "h-[calc(100vh-70px)]",
+          "items-center justify-center text-center",
+        )}
+      >
+        <div className={clsx(`${semiBoldFont.className}`, "text-7xl")}>
+          Typesafe Travellers
+        </div>
+
+        <div className={clsx("text-2xl", `${regularFont.className}`)}>
+          This is a website for people who want to travel the world, but do not
```
Comment (AyanavaKarmakar) [inline src/pages/about.tsx]: Welcome to our group trip-planning app! We are a team of travel enthusiasts who understand the challenges of planning a group trip. We created this app to make the process as easy and stress-free as possible.

Our app allows users to create a trip itinerary, share it with their group, and collaborate on details such as accommodations, transportation, and activities.

We believe that group trip planning should be fun and exciting, not overwhelming. That's why we've designed our app to be user-friendly and intuitive.

Thank you for choosing our group trip-planning app. Happy travels!

File: src/pages/about.tsx [hunk 2]
```diff
@@ -1,5 +1,64 @@
...
+          Next.js, TypeScript, Tailwind CSS, NextAuth.js, PlanetScale,
+          Heroicons, Radix UI, Radix Icons, MySQL, Prisma, Repl.it, tRPC, Zod
+        </div>
+
+        <div className={clsx("text-2xl", `${regularFont.className}`)}>
+          Made with &hearts; using the T3 stack
+        </div>
+
+        <footer className={clsx(`${regularFont.className}`, "text-xl")}>
+          &copy; 2023 Typesafe Travellers
```
Comment (AyanavaKarmakar) [inline src/pages/about.tsx]:
```suggestion
          &copy; {new Date().getFullYear()} Typesafe Travellers
```

File: src/pages/about.tsx [hunk 3]
```diff
@@ -1,5 +1,64 @@
...
+        </div>
+
+        <div className={clsx("text-2xl", `${regularFont.className}`)}>
+          <h1 className={clsx(`${semiBoldFont.className}`)}>Tech Stack</h1>
+          Next.js, TypeScript, Tailwind CSS, NextAuth.js, PlanetScale,
+          Heroicons, Radix UI, Radix Icons, MySQL, Prisma, Repl.it, tRPC, Zod
+        </div>
+
+        <div className={clsx("text-2xl", `${regularFont.className}`)}>
+          Made with &hearts; using the T3 stack
```
Comment (AyanavaKarmakar) [inline src/pages/about.tsx]:
```suggestion
          Made with &hearts; by TypeSafe Travellers using the T3 stack
```

**Review by AyanavaKarmakar — CHANGES_REQUESTED** [review]

File: src/pages/about.tsx [hunk 4]
```diff
@@ -1,5 +1,63 @@
...
+        </div>
+
+        <div className={clsx("text-2xl", `${regularFont.className}`)}>
+          Welcome to our group trip-planning app! We are a team of travel
+          enthusiasts who understand the challenges of planning a group trip. We
+          created this app to make the process as easy and stress-free as
+          possible. Our app allows users to create a trip itinerary, share it
+          with their group, and collaborate on details such as accommodations,
+          transportation, and activities. We believe that group trip planning
+          should be fun and exciting, not overwhelming. That is why we have
```
Comment (AyanavaKarmakar) [inline src/pages/about.tsx]:
```suggestion
          should be fun and exciting, not overwhelming. That&apos;s why we&apos;ve
```

**Review by AyanavaKarmakar — CHANGES_REQUESTED** [review]

**Comment by AyanavaKarmakar** [comment]
Updated

image

**Review by AyanavaKarmakar — APPROVED** [review]

File: src/pages/about.tsx [hunk 5]
```diff
@@ -1,5 +1,181 @@
...
+            className={clsx(
+              "lg:text-2xl",
+              "text-justify",
+              `${regularFont.className}`,
+            )}
+          >
+            Welcome to our group trip-planning app! We are a team of travel
+            enthusiasts who understand the challenges of planning a group trip.
+            We created this app to make the process as easy and stress-free as
+            possible. Our app allows users to create a trip itinerary, share it
```
Comment (Nishith-Savla) [inline src/pages/about.tsx]:
```suggestion
            possible. Our web app allows users to create a trip itinerary, share it
```

File: src/pages/about.tsx [hunk 6]
```diff
@@ -1,5 +1,181 @@
...
+            with their group, and collaborate on details such as accommodations,
+            transportation, and activities. We believe that group trip planning
+            should be fun and exciting, not overwhelming. That&apos;s why
+            we&apos;ve designed our app to be user-friendly and intuitive. Thank
+            you for choosing our group trip-planning app. Happy travels!
```
Comment (Nishith-Savla) [inline src/pages/about.tsx]: What about this?

```suggestion
            you for choosing our group trip-planning app. Bon voyage!
```

File: src/pages/about.tsx [hunk 7]
```diff
@@ -1,5 +1,181 @@
...
+          <footer
+            className={clsx(
+              `${regularFont.className}`,
+              "text-xl lg:text-xl",
+              "pt-6 lg:pt-10",
+              "flex flex-col gap-0",
+            )}
+          >
+            <div className={clsx("text-2xl")}>
+              &copy; {new Date().getFullYear()}{" "}
```
Comment (Nishith-Savla) [inline src/pages/about.tsx]:
```suggestion
              {`& ${new Date().getFullYear()} `}
```

**Review by Nishith-Savla — APPROVED** [review]

**Pull request merged**